mir_connector/inorbit_mir_connector/src/mir_api/__init__.py

```diff
@@ -1,3 +1,8 @@
 from .mir_api_base import MirApiBaseClass  # noqa: F401
 from .mir_api_v2 import MirApiV2  # noqa: F401
 from .mir_api_v2 import MirWebSocketV2  # noqa: F401
+
+# Available API versions
+APIS = {
+    "v2.0": {"rest": MirApiV2, "ws": MirWebSocketV2},
+}
```

Comment:
Without having much context, are we sure these versions are fixed?
E.g:  V2.0 means V2 for Rest and WS?  
Could it happen that a model uses different APIs versions for WS and rest?